### DISCUSSION TITLE
Close Buffers not Tabs - Tabufline Plugin

### DISCUSSION THREAD

**Discussion by zwk5131** [NvChad/NvChad#2657] [Q&A] [answered]
Hello :D

I have a wired behavior when working with Tabufline...
I worked with 2 buffers then added a new tab in this new tab i created 3 buffers.
Now i tried to delete a Buffer in tab 2 with the command `:bd` and it closes all buffers in tab 2...
Is there something wrong or is another command needed ?

**Comment by siduck** [comment, marked as answer]
thats normal nvim behavior @zwk5131 , so please dont keep 1 buffer opened in multiple tabs, closing it will close those tabs who only had 1 buffer (that buffer which u were trying to close)

**Reply by zwk5131** [reply]
They were all different buffers and I didn't had a buffer in different tabcontainers.. I didn't closed the first one i closed the 2nd buffer of tabcontainer 2. Anyway if there is not a special command to quit only a buffer and not the hole tabcontainer.... I would like to disable tabs is that possible ?

**Reply by siduck** [reply]
screen record it. i cant reproduce it here

**Reply by zwk5131** [reply]
Thx for taking time :) Here it is...
 
https://github.com/NvChad/NvChad/assets/149153490/dd04a7fa-ac71-4661-9911-2873e058ee74

When I click on X with mouse on guitabpage dsds.md it works fine

I'm on ubuntu... i tried a clean install nvchad in a vm and the exact same happens...

**Reply by zwk5131** [reply]
@siduck 
Maybe the problem is the different buffernumbers... i get always different buffer numbers... which may is confusing for the tabcontainers...
`:echo bufnr('%')`

Or did you find already something ?

**Reply by siduck** [reply]
@zwk5131 i'll look into this , this saturday :)
